src/test/java/move/FoMoveBehaviorTest.java

```diff
@@ -0,0 +1,90 @@
+package move;
+
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import piece.Piece;
+import piece.PieceType;
+import piece.Pieces;
+import piece.Position;
+import piece.Route;
+import piece.Team;
+
+public class FoMoveBehaviorTest {
+
+    @Test
+    void 포는_가능한_경로를_반환한다() {
+        // given
+        Position startPosition = new Position(0, 0);
+        Position endPosition = new Position(5, 0);
+        MoveBehavior moveBehavior = new FoMoveBehavior();
+
+        // when
+        Route route = moveBehavior.getLegalRoute(startPosition, endPosition, Team.BLUE);
+
+        // then
+        List<Position> positions = route.positions();
+        Assertions.assertThat(positions.size()).isEqualTo(5);
```

Comment:
```suggestion
        Assertions.assertThat(positions).hasSize(5);
```
`assertAll`이라는 키워드에 대해서 학습해볼까요?